tests/testthat/test-ppc-intervals.R

```diff
@@ -1,7 +1,7 @@
 library(bayesplot)
 context("PPC: intervals & ribbon")
 
-source("data-for-ppc-tests.R")
+source(test_path("data-for-ppc-tests.R"))
```

Comment:
I can't believe I hadn't seen the `testthat::test_path` function before! Much better than having to manually change the path when working locally with the file.